apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -115,6 +133,85 @@ export default function CourseOfferingEditor(props) {
           onChange={e => updateCourseOffering('assignable', e.target.checked)}
         />
       </label>
+      <h2>{`Curriculum Catalog Settings`}</h2>
```

Comment:
Nit: You can just have `<h2>Curriculum Catalog Settings</h2>` here

Reply:
lol good catch TY!